Task3_Products/routers/categories.py

```diff
@@ -0,0 +1,72 @@
+from typing import Literal, Optional
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from Task3_Products import crud, schemas, database
```

Comment:
Just for notice, no need to fix: better to import exact objects you need (for example `from Task3_Products.schemas import Category` to not import unnecessary objects.